app/src/test/resources/.env.test

```diff
@@ -0,0 +1,3 @@
+ENV=local
```

Comment:
Is this file needed? I don't see it being loaded in the tests anywhere.

Reply:
File got deleted. Good one!